set.mm

```diff
@@ -659185,6 +659196,173 @@ D Fn ( I ... ( M - 1 ) ) /\
+       is symmetric, we can interchange ` X ` and ` Y ` in the result.
+       (Contributed by SN, 29-Jan-2025.) $)
+    grpcominv1 $p |- ( ph -> ( X .+ ( N ` Y ) ) = ( ( N ` Y ) .+ X ) ) $=
+      ( cfv co wceq grpassd 3eqtr4rd wcel grpinvcld eqid grplinvd oveq1d eqtr2d
+      c0g sn-grplidd oveq2d grpasscan2d cgrp wb grpcld grprcan syl13anc mpbid )
+      AFGEOZCPZGCPZUPFCPZGCPZQZUQUSQZAUPFGCPZCPZFUTURAUPGCPZFCPZUPGFCPZCPFVDABC
+      DUPGFHIKABDEGHJKMUAZMLRAVFDUFOZFCPFAVEVIFCABCDEGVIHIVIUBZJKMUCUDABCDFVIHI
+      VJKLUGUEAVCVGUPCNUHSABCDUPFGHIKVHLMRABCDEFGHIJKLMUISADUJTUQBTUSBTGBTVAVBU
+      KKABCDFUPHIKLVHULABCDUPFHIKVHLULMBCDUQUSGHIUMUNUO $.
+  $}
+
+  ${
+    odm1inv.x $e |- X = ( Base ` G ) $.
+    odm1inv.o $e |- O = ( od ` G ) $.
+    odm1inv.t $e |- .x. = ( .g ` G ) $.
+    odm1inv.i $e |- I = ( invg ` G ) $.
+    odm1inv.g $e |- ( ph -> G e. Grp ) $.
+    odm1inv.1 $e |- ( ph -> A e. X ) $.
+    $( The (order-1)th multiple of an element is its inverse.  (Contributed by
+       SN, 31-Jan-2025.) $)
+    odm1inv $p |- ( ph -> ( ( ( O ` A ) - 1 ) .x. A ) = ( I ` A ) ) $=
+      ( cfv co c1 wcel wceq eqid syl csg c0g cmin odid mulg1 oveq12d cgrp odcld
+      cz nn0zd 1zzd mulgsubdir syl13anc grpinvval2 syl2anc 3eqtr4d ) ABFNZBCOZP
+      BCOZDUANZOZDUBNZBUTOZUQPUCOBCOZBENZAURVBUSBUTABGQZURVBRMBCDFGVBHIJVBSZUDT
+      AVFUSBRMGCDBHJUETUFADUGQZUQUIQPUIQVFVDVARLAUQABGDFHIMUHUJAUKMGCDUQUTPBHJU
+      TSZULUMAVHVFVEVCRLMGDUTEBVBHVIKVGUNUOUP $.
+  $}
+
+  ${
+    $d S a $.  $d G a $.  $d ph a $.
+    finodsubmsubg.o $e |- O = ( od ` G ) $.
+    finodsubmsubg.g $e |- ( ph -> G e. Grp ) $.
+    finodsubmsubg.s $e |- ( ph -> S e. ( SubMnd ` G ) ) $.
+    finodsubmsubg.1 $e |- ( ph -> A. a e. S ( O ` a ) e. NN ) $.
+    $( A submonoid whose elements have finite order is a subgroup.
+       (Contributed by SN, 31-Jan-2025.) $)
+    finodsubmsubg $p |- ( ph -> S e. ( SubGrp ` G ) ) $=
+      ( cfv wcel wral wa co cmg wceq cbs eqid syl ad2antrr csubmnd cminusg cmin
+      csubg cv cn c1 cgrp adantr wss submss sselda odm1inv cmnd submmnd nnm1nn0
+      cress adantl ressbas2 eleqtrd mulgnn0cl syl3anc submmulg 3eltr4d eqeltrrd
+      cn0 simplr ex ralimdva mpd wb issubg3 mpbir2and ) ABCUDJKZBCUAJKZEUEZCUBJ
+      ZJZBKZEBLZHAVPDJZUFKZEBLVTIAWBVSEBAVPBKZMZWBVSWDWBMZWAUGUCNZVPCOJZNZVRBWD
+      WHVRPWBWDVPWGCVQDCQJZWIRZFWGRZVQRZACUHKZWCGUIABWIVPAVOBWIUJZHWIBCWJUKSZUL
+      UMUIWEWFVPCBUQNZOJZNZWPQJZWHBWEWPUNKZWFVFKZVPWSKWRWSKAWTWCWBAVOWTHBWPCWPR
+      ZUOSTWBXAWDWAUPURZWEVPBWSAWCWBVGZABWSPZWCWBAWNXEWOBWIWPCXBWJUSSTZUTWSWQWP
+      WFVPWSRWQRZVAVBWEVOXAWCWHWRPAVOWCWBHTXCXDBWGWQCWPWFVPWKXBXGVCVBXFVDVEVHVI
+      VJAWMVNVOVTMVKGEBCVQWLVLSVM $.
+  $}
+
+  ${
+    $d .0. a $.  $d G a $.
+    sn-0subg.z $e |- .0. = ( 0g ` G ) $.
+    $( A shorter proof of ~ 0subg using ~ df-od .  (Contributed by SN,
+       31-Jan-2025.) $)
+    sn-0subg $p |- ( G e. Grp -> { .0. } e. ( SubGrp ` G ) ) $=
```

Comment:
If no extra axioms are used,  then this should probably be 0subg, and the current 0subg could be renamed 0subgALT, or maybe removed (that is, deprecated).

Reply:
there is an ordering issue where 0subg is used in 0nsg and idressubgsymg, which comes before df-od, so sn-0subg can't easily be moved in

Reply:
Then maybe you can bring your version as 0subgALT to Main, after the theorems you need. I think your version deserves to be in Main.